documentation/docs/25-build-and-deploy/90-adapter-vercel.md

```diff
@@ -11,33 +11,65 @@ This adapter will be installed by default when you use [`adapter-auto`](adapter-
-			edge: false,
-
-			// an array of dependencies that esbuild should treat
-			// as external when bundling functions. this only applies
-			// to edge functions, and should only be used to exclude
-			// optional dependencies that will not run outside Node
-			external: [],
-
-			// if true, will split your app into multiple functions
-			// instead of creating a single one for the entire app
-			split: false
+			// see the section on deployment configuration below
+			runtime: 'nodejs18.x',
+			regions: ['iad1'], // if runtime is `edge`, this defaults to `all`
+			memory: 1024,
+			maxDuration: undefined, // 5s for Hobby plans, 15s for Pro, 30s for Enterprise
+			split: false,
+			external: []
 		})
 	}
 };
 ```
 
+## Deployment configuration
+
+To control how your routes are deployed to Vercel as functions, you can specify deployment configuration, either through the `defaultConfig` option shown above or with [`export const config`](/docs/page-options#config) inside `+server.js`, `+page(.server).js` and `+layout(.server).js` files.
+
+For example you could deploy some parts of your app as [Edge Functions](https://vercel.com/docs/concepts/functions/edge-functions)...
+
+```js
+/// file: about/+page.js
+/** @type {import('@sveltejs/adapter-vercel').Config} */
+export const config = {
+	runtime: 'edge'
+};
+```
+
+...and others as [Serverless Functions](https://vercel.com/docs/concepts/functions/serverless-functions) (note that by specifying `config` inside a layout, it applies to all child pages):
+
+```js
+/// file: admin/+layout.js
+/** @type {import('@sveltejs/adapter-vercel').Config} */
+export const config = {
+	runtime: 'nodejs18.x'
+};
+```
+
+The following options apply to all functions:
+
+- `runtime`: `'edge'`, `'nodejs16.x'` or `'nodejs18.x'` (the default)
+- `regions`: an array of [edge network regions](https://vercel.com/docs/concepts/edge-network/regions) (defaulting to `["iad1"]` for serverless functions) or `'all'` if `runtime` is `edge` (its default)
+- `split`: if `true`, causes a route to be deployed as an individual function. If `split` is set to `true` at the adapter level, all routes will be deployed as individual functions
+
+Additionally, the following options apply to edge functions:
+- `envVarsInUse`: an array of environment variables that should be accessible inside the edge function
+- `external`: an array of dependencies that esbuild should treat as external when bundling functions. This should only be used to exclude optional dependencies that will not run outside Node
+
+And the following option apply to serverless functions:
+- `memory`: the amount of memory available to the function. Defaults to `1024` Mb, and can be [increased](https://vercel.com/docs/concepts/limits/overview#serverless-function-memory) up to `3008` on Pro or Enterprise accounts
+- `maxDuration`: maximum execution duration of the function. Defaults to `5` seconds for Hobby accounts, `15` for Pro and `30` for Enterprise
```

Comment:
This seems to come from the Vercel V1 API docs which are outdated, see https://vercel.com/docs/concepts/limits/overview for serverless timeout duration

Reply:
good catch, thank you!